modules/installation-osp-kuryr-api-scaling.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-configure{context}"]
+= Scaling API using {rh-openstack} Octavia
```

Comment:
Scaling which API? :) @luis5tb

Reply:
@luis5tb Since modules are (in theory) supposed to be able to stand alone, what's a good title for this?

`Scaling up API availability for clusters that run on {rh-openstack}` 

Is identifying Octavia in the title important, given that it will be in the module's prerequisites and body text?

Reply:
Well, it is not for availability, but for scalability. And I highlighted Octavia since it is what we are using to provide such scalability. There are other methods probably taht could be used without relying on Octavia OSP component, but this is about using Octavia to provide that increase scalability

Reply:
👍